probe-rs/src/bin/probe-rs/cmd/info.rs

```diff
@@ -138,13 +136,13 @@ fn try_show_info(
         match probe.try_into_xtensa_interface() {
             Ok(mut interface) => {
                 if let Err(e) = show_xtensa_info(&mut interface) {
-                    log::warn!("Error showing Xtensa chip information: {}", e);
+                    log::warn!("Error showing Xtensa chip information: {:?}", anyhow!(e));
                 }
 
                 probe = interface.close();
             }
             Err((interface_probe, e)) => {
-                print_error(&e);
+                log::warn!("Error showing Xtensa chip information: {:?}", anyhow!(e));
```

Comment:
I am doubtful this prints the whole error stack, which is what we want.

Also, I don't believe actual CLI output should be part of the logging :) I would prefer if this was print! based. I am happy to add colors tho.

Logging for me is always on top of CLI output, not something to replace it.

Reply:
> I am doubtful this prints the whole error stack, which is what we want.

[It actually does!](https://docs.rs/anyhow/latest/anyhow/struct.Error.html#display-representations)

Easy enough to change from log to print

Reply:
but to be fair, since `probe-rs info` doesn't know if something is an actual error or just a device isn't what it's trying to read from, I'm thinking these errors aren't as important... this is actually kind of hard to figure out

Reply:
Ohhhh, I was scammed my entire life! I always implemented this myself LOL.